pkgs/tools/backup/hpe-ltfs/default.nix

```diff
@@ -13,10 +13,12 @@ stdenv.mkDerivation rec {
 
   sourceRoot = "source/ltfs";
 
+  patches = [ ./remove-sysctl.patch ];
```

Comment:
do you mind adding a comment as to why this necessary. Essentially your PR description would suffice.

Reply:
done :heavy_check_mark: